### PR TITLE
Auto migrate from old schema by checking autoMigrateOldSchema option

### LINKED ISSUE #82: Shouldn't db:migrate detect old_schema and convert table automatically?
Since we and many more of us have auto-deploy on green, `sequelize db:migrate:old_schema` command doesn't seems to be a nice approach. Specially for those (our case) who have servers running and protected against manual access.

Shouldn't `db:migrate` task check if it needs to migrate from `old_schema` and migrate automatically?

ping @sdepold @mickhansen

### PR DESCRIPTION
**Pull request by cusspvz**
closes #82

### CODE REVIEW

**Comment by cusspvz** [comment]
Ready for code review! Hope to be everything as you expected. :)

**Comment by cusspvz** [comment]
Seems that test on sqlite dialect is intermittent. Could you please check what could be the cause?

**Comment by sdepold** [comment]
Let me check

**Comment by sdepold** [comment]
Hm it works nicely on my local machine ... I just triggered another build for sqlite

**Comment by sdepold** [comment]
Works :)

File: test/db/migrate/old_schema.test.js [hunk 1]
```diff
@@ -54,9 +56,27 @@ var gulp    = require("gulp");
...
     });
 
+    describe(Support.getTestDialectTeaser(flag + " auto migrate old schema"), function () {
+      beforeEach(function (done) {
+        return prepareLegacyScenario.call(this, {
+          autoMigrateOldSchema: true
+        }, function () { done(); });
+      });
+
+      it("shouldn't enforce if we provide autoMigrateOldSchema option", function (done) {
```
Comment (sdepold) [inline test/db/migrate/old_schema.test.js]: Could you please add a check before and after the migration, that ensures the previously old table structure is now the new one?

**Comment by sdepold** [comment]
Besides the little note about the test everything looks good :) Thanks a lot!

**Comment by cusspvz** [comment]
> Hm it works nicely on my local machine ... I just triggered another build for sqlite

Locally it worked flawless also, but since it has failed on Travis, I've supposed that it was intermittent.

> Besides the little note about the test everything looks good :) Thanks a lot!

Nice to ear that! I've already added checks for SequelizeMeta table schema.
Thank you for making sequelize! :)

Ready to merge.

**Comment by sdepold** [comment]
nice!

**Comment by sdepold** [comment]
just released 1.0.5

**Comment by cusspvz** [comment]
That was fast as hell!!!
Thank you once again! :D

**Comment by sdepold** [comment]
Thank you! btw. I added some words about that topic to the readme file

**Pull request merged**